Automattic/wp-calypso
# Sites Dashboard: Update details for sites pending migration

Related to #95167

This branch is based on `add/migration-status-to-sites` and will require a rebase once that branch gets merged into trunk.

## Proposed Changes

* Add a custom migration icon for incoming sites.
* Replace Site Title with `Incoming migration` for sites that don’t have a name yet.

![image](https://github.com/user-attachments/assets/cd65a28d-ecba-4779-86d2-6710a931fadc)


## Why are these changes being made?


* This is part of the following project: pfuQfP-Hj-p2.

## Testing Instructions



* Go to `/setup/migration`, checkout, and select "I'll do it myself".
* Take note of the site domain generated. You can find it in the URL.
* Go to `/sites` and search for the generated site domain.
* Make sure the site title is updated to `Incoming migration` and the icon is updated as per the design.

## Pre-merge Checklist
